job_templates/sag_nemo/app_1/config_fed_client.conf

```diff
@@ -0,0 +1,81 @@
+{
+  # version of the configuration
+  format_version = 2
+
+  # This is the application script which will be invoked. Client can replace this script with user's own training script.
+  app_script = "megatron_gpt_peft_tuning.py"
+
+  # Additional arguments needed by the training code. For example, in lightning, these can be --trainer.batch_size=xxx.
+  app_config = ""
+
+  # Additional arguments needed by DDP.
+  ddp_config = "--nnodes=1 --nproc_per_node=1 --master_port=7777"
+
+  # Client Computing Executors.
+  executors = [
+    {
+      # tasks the executors are defined to handle
+      tasks = ["train"]
+
+      # This particular executor
+      executor {
+
+        # Eexecutor name : PTClientAPILauncherExecutor
```

Comment:
some typos in this conf, please refer to this PR to fix them:
https://github.com/NVIDIA/NVFlare/pull/2101

Reply:
Thanks. Fixed.